lib/libutee/tee_api_property.c

```diff
@@ -516,15 +554,21 @@ TEE_Result TEE_GetPropertyAsBinaryBlock(TEE_PropSetHandle propsetOrEnumerator,
 		goto err;
 	}
 
-	val = pv.u.str_val;
-	val_len = strlen(val);
-	size = *valueBufferLen;
-	if (!base64_dec(val, val_len, valueBuffer, &size)) {
```

Comment:
This will break the test for properties in `optee_test/ta/os_test`, myprop.binaryblock